x-pack/plugins/secops/public/components/timeline/data_providers/data_provider.ts

```diff
@@ -18,9 +18,12 @@ export interface DataProvider {
    */
   enabled: boolean;
   /**
-   * Returns a query that, when executed, returns the data for this provider
+   * Returns a Component query that, when executed, returns the data for this provider
    */
-  getQuery: () => string;
+  // tslint:disable-next-line:no-any
```

Comment:
This line isn't needed it looks like. Thanks for committing to keeping the file free of `any` as much as possible!